test/proto/bookstore.proto

```diff
@@ -0,0 +1,159 @@
+syntax = "proto3";
+
+package bookstore;
```

Comment:
This seems a rather complicated proto for doing tests, can't we express everything we need to test in a much smaller proto?